v2_migrations/2024-08-28-081838_update_v2_primary_key_constraints/up.sql

```diff
@@ -88,12 +88,21 @@ ADD PRIMARY KEY (id);
 ------------------------ Payment Intent -----------------------
 ALTER TABLE payment_intent DROP CONSTRAINT payment_intent_pkey;
 
+-- To work around database constraints
+UPDATE payment_intent
+SET id = payment_id
+WHERE id IS NULL;
+
 ALTER TABLE payment_intent
 ADD PRIMARY KEY (id);
 
 ------------------------ Payment Attempt -----------------------
 ALTER TABLE payment_attempt DROP CONSTRAINT payment_attempt_pkey;
 
+UPDATE payment_attempt
```

Comment:
we should not do this work around, this work around will fail with all the logic.